### ISSUE TITLE
make start fails with image verification error

### ISSUE THREAD

**Issue by mcastelino** [intel/pmem-CSI#295]
# make start fails with image verification error

```
Downloading clear-29690-cloud.img.xz image
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 54.8M  100 54.8M    0     0  3856k      0  0:00:14  0:00:14 --:--:-- 6320k
Verification failure
140168122532288:error:21075075:PKCS7 routines:PKCS7_verify:certificate verify error:../crypto/pkcs7/pk7_smime.c:285:Verify error:unsupported certificate purpose
Image verification failed, see error above.

"unsupported certificate purpose" is a known issue caused by an incompatible openssl
version (https://github.com/clearlinux/distribution/issues/85). Other errors might indicate
a download error or man-in-the-middle attack.

To skip image verification run:
TEST_CHECK_SIGNED_FILES=false make start
test/start-stop.make:17: recipe for target 'start' failed
make: *** [start] Error 2
```

# Environment

```bash
$ cat /etc/lsb-release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=18.04
DISTRIB_CODENAME=bionic
DISTRIB_DESCRIPTION="Ubuntu 18.04.2 LTS"
```

```bash
$docker info
Server Version: 18.09.6
Storage Driver: overlay2
...
containerd version: bb71b10fd8f58240ca47fbb579b9d1028eea7c84
runc version: 2b18fe1d885ee5083ef9f0838fee39b62d653e30
init version: fec3683
Security Options:
 seccomp
  Profile: default
Kernel Version: 5.0.11-050011-generic
Operating System: Ubuntu 18.04.2 LTS
OSType: linux
Architecture: x86_64
```

**Comment by mcastelino** [comment]
https://github.com/clearlinux/distribution/issues/85#issuecomment-498424419

**Comment by mcastelino** [comment]
Workaround for now

```
TEST_CHECK_SIGNED_FILES=0 make -d start
```

**Comment by pohly** [comment]
This is an issue that I brought up with the Clear Linux team a while ago in https://github.com/clearlinux/distribution/issues/85#issuecomment-395354903 but that was closed without resolving that particular aspect of the problem.

We tried  to make the error message as obvious as possible. Do we need to improve it further? Or should we disable the check by default? But then we might as well just remove the check entirely, because no-one will remember to turn it on.
